geps/overview.md

```diff
@@ -66,6 +66,11 @@ meeting before merging. Most GEPS will proceed through the following states:
 
 * **Provisional:** The goals described by this GEP have consensus but
   implementation details have not been agreed to yet.
+* **Prototyping:** An extension of `Provisional` which can be opted in to in
+  order to indicate to the community that there are some active practical tests
+  and experiments going on which are intended to be a part of the development
+  of this GEP. This may include APIs or code, but that content _must_ not be
+  distributed with releases.
```

Comment:
This new proposed status feels like it's representative of the current state of some GAMMA GEPs.

/cc @howardjohn @keithmattix @mikemorris

Reply:
+1 - I like Prototyping better than Provisional